src/meta/processors/jobMan/JobManager.h

```diff
@@ -79,7 +101,15 @@ class JobManager : public nebula::cpp::NonCopyable, public nebula::cpp::NonMovab
     void removeExpiredJobs(const std::vector<std::string>& jobKeys);
 
 private:
-    std::unique_ptr<folly::UMPSCQueue<int32_t, true>>  queue_;
+    // Todo(pandasheep)
+    // When folly is upgraded, PriorityUMPSCQueueSet can be used
+    // Use two queues to simulate priority queue, Divide by job cmd
+    std::unique_ptr<folly::UMPSCQueue<JobID, true>>    lowPriorityQueue_;
+    std::unique_ptr<folly::UMPSCQueue<JobID, true>>    highPriorityQueue_;
+
+    // The job in running or queue
+    folly::ConcurrentHashMap<JobID, JobDescription>    jobMap_;
```

Comment:
may be we can called this 'inFlightJobs' ?

Reply:
Good point, 👍